geoips/commandline/ancillary_info/cmd_instructions.yaml

```diff
@@ -289,6 +289,20 @@ instructions:
   #     `geoips test unit-test -p <package_name> <directory_name> <-n> <script_name>`
   #   output_info:
   #     - Output of the Unit Test Being Ran
+  geoips_tree:
+    help_str: |
+      Print a tree of all of the available GeoIPS CLI commands. This will display every
+      command in a top-down tree which gives a visual structure of how the CLI can be
+      ran. <--max-depth> must be greater than or equal to 1, where depth denotes the
+      level of the command you'd like to display. Ie. <0> geoips <1> list <2> scripts.
+    usage_str: |
+      To use, type `geoips tree <--max-depth> <int> <--color> <--short-name>`.
+    output_info:
+      - The tree of available GeoIPS CLI commands up to '--max-depth' levels, where
+        depth denotes the level of the command you'd like to display.
+        Ie. <0> geoips <1> list <2> scripts.
```

Comment:
Nitpick: "Ie." should have two periods in it like this: "I.e.". It stands for "id est" which is Latin for "that is".

Reply:
Fixed!